tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp

```diff
@@ -694,8 +694,15 @@ void test_dimensionful_spin_magnitude(
   const double spin_magnitude = StrahlkorperGr::dimensionful_spin_magnitude(
```

Comment:
can you put the test and the void function into the same commit? I think this just means squashing the first two commits together.